docs/src/use/configure/ignore.md

```diff
@@ -8,135 +8,76 @@ eleventyNavigation:
-
-  ```text
-  # Allowlist 'test.js' in the '.build' folder
-  # But do not allow anything else in the '.build' folder to be linted
-  !.build
-  .build/*
-  !.build/test.js
-  ```
-
-  The following `--ignore-pattern` is also equivalent:
-
-  ```shell
-  eslint --ignore-pattern '!.build' --ignore-pattern '.build/*' --ignore-pattern '!.build/test.js' parent-folder/
-  ```
-
-## Using an Alternate File
-
-If you'd prefer to use a different file than the `.eslintignore` in the current working directory, you can specify it on the command line using the `--ignore-path` option. For example, you can use `.jshintignore` file because it has the same format:
-
-```shell
-eslint --ignore-path .jshintignore file.js
-```
-
-You can also use your `.gitignore` file:
+You can also unignore files on the command line using `--ignore-pattern`, such as:
 
 ```shell
-eslint --ignore-path .gitignore file.js
+npx eslint . --ignore-pattern "!node_modules/"
 ```
 
-Any file that follows the standard ignore file format can be used. Keep in mind that specifying `--ignore-path` means that the existing `.eslintignore` file is not used. Note that globbing rules in `.eslintignore` follow those of `.gitignore`.
-
-## Using eslintIgnore in package.json
+## Glob Pattern Resolution
 
-If an `.eslintignore` file is not found and an alternate file is not specified, ESLint looks in `package.json` for the `eslintIgnore` key to check for files to ignore.
+How glob patterns are evaluated depends on where they are located and how they are used:
 
-```json
-{
-    "name": "mypackage",
-    "version": "0.0.1",
-    "eslintConfig": {
-        "env": {
-            "browser": true,
-            "node": true
-        }
-    },
-    "eslintIgnore": ["hello.js", "world.js"]
-}
-```
+1. When using `ignores` in an `eslint.config.js` file, glob patterns are evaluated relative to the `eslint.config.js` file.
+1. When using `ignores` in an alternate configuration file specified using the `--config` command line option, glob patterns are evaluated relative to the current working directory.
+1. When using `--ignore-pattern`, glob patterns are evaluated relative to the current working directory.
 
 ## Ignored File Warnings
 
-When you pass directories to ESLint, files and directories are silently ignored. If you pass a specific file to ESLint, then ESLint creates a warning that the file was skipped. For example, suppose you have an `.eslintignore` file that looks like this:
+When you pass directories to the ESLint CLI, files and directories are silently ignored. If you pass a specific file to ESLint, then ESLint creates a warning that the file was skipped. For example, suppose you have an `eslint.config.js` file that looks like this:
 
-```text
-foo.js
+```js
+// eslint.config.js
+export default [
+    {
+        ignores: ["foo.js"]
+    }
+]
 ```
 
 And then you run:
```

Comment:
For consistency with other examples, it might be good to use `npx` in this one too (`eslint foo.js` -> `npx eslint foo.js` on line 86; GitHub doesn't let me leave a suggestion there).